tests/tests_interlocking/test_interlocking_configuration.py

```diff
@@ -1,8 +1,55 @@
-from src.interlocking_component.route_controller import InterlockingConfiguration
+from uuid import UUID
 
+import pytest
 
-def test_default_true():
-    """This test if an InterlockingConfiguration Object can be initialized
-    and if the default for dynamicRouting is false."""
-    interlocking_configuration = InterlockingConfiguration()
-    assert not interlocking_configuration.dynamicRouting
+from src.interlocking_component.interlocking_configuration import (
+    InterlockingConfiguration,
+)
+from tests.decorators import recreate_db_setup
+
+
+@pytest.mark.parametrize(
+    "object_as_dict",
+    [
+        {
+            "dynamicRouting": True,
+        },
+    ],
+)
+class TestCorrectFilledDict:
+    """Test that a object of a class can be created and deserialized with valid data."""
+
+    @recreate_db_setup
+    def setup_method(self):
+        pass
+
+    def test_create(self, object_as_dict: dict):
+        """Test that a object of a class can be created."""
+        obj = InterlockingConfiguration.create(
+            **object_as_dict,
+        )
+        assert (
+            InterlockingConfiguration.select()
+            .where(InterlockingConfiguration.id == obj.id)
+            .first()
+            == obj
+        )
+
+    def test_serialization(self, object_as_dict: dict):
+        """Test that an object of a class can be serialized."""
+        obj = InterlockingConfiguration.create(
+            **object_as_dict,
+        )
+
+        for key in object_as_dict.keys():
+            assert getattr(obj, key) == object_as_dict[key]
+
+    def test_deserialization_full_dict(self, object_as_dict: dict):
+        """Test that an object of a class can be deserialized."""
+        obj = InterlockingConfiguration.Schema().load(
```

Comment:
This works completely fine. But is there a reason not to use `InterlockingConfiguration.from_dict` ? 🤔

Reply:
I have changed it. The only reason was copy and paste. :wink: